uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js

```diff
@@ -433,6 +474,34 @@ define([
       }
 
       /**
+       * Pick a specific leave request day type
+       *
+       * @param {string} value - The leave request day type value to match
+       * @return {object}
+       */
+      function getStatusFromValue(value) {
```

Comment:
I'm getting confused by this function

1. The documentation says "Pick a specific leave request day type", but you are using `vm.leaveRequestStatuses`?
2. This function seems extremely similar to `valueOfRequestStatus()`. Can you tell me what the difference is?